code/src/Components/Form.js

```diff
@@ -0,0 +1,51 @@
+import React, {useState} from "react";
+import "styles/form.css";
+import Input from "./Input";
+import Select from "./Select";
+import Radiobuttons from "./Radiobuttons";
+import Summary from "./Summary";
+
+
+const Form = () => {
+    
+    /* State hooks */
+    const [name, setName] = useState('');
+    const [numberOfPlants, setNumberOfPlants] = useState(''); 
+    const [favoritePlant, setfavoritePlant] = useState('');
+    const [showSummary, setShowSummary] = useState(false);
+    
+    /* Functions for handling input from form */ 
+    const handleNameInput = (inputName) => {
+        setName(inputName);
+    };
```

Comment:
Tip: when you only have one argument in a function you can omit the parenthesis.  The function would then look like: 

```
const handleNameInput = inputName => {
          setName(inputName);
},
```